src/main/groovy/com/instabot/InstaBot.groovy

```diff
@@ -21,7 +21,10 @@ class InstaBot {
     def start(String loginUsername, String loginPassword) throws InterruptedException {
         try {
             LOG.info("Start InstaBot execution")
+            // TODO It's kind of weird that you instantiate the instaWebDriver here, then pass it as param and then just close it.
+            // I would extract the following and followers here, close the connection ASAP and pass them as parameters to RelatedUsersUpdater
```

Comment:
Extracting and updating the relationship with users is part of the same process; any of these two components are not going to be used for other purposes. Are you sure that it is better to separate them?